src/include/utime.h

```diff
@@ -38,23 +38,24 @@
 class utime_t {
 public:
   struct {
-    __u32 tv_sec, tv_nsec;
+    time_t tv_sec;
+    long tv_nsec;
```

Comment:
we can't change the on disk encoding without breaking compatibility

Reply:
+1